### ISSUE TITLE
Implement non-strict mode

### ISSUE THREAD

**Issue by renskiy** [vkryachko/tasklocals#3]
Non-stric mode is mode when attempting to get/set value from/to `local` instance silently ignores lack of the current task and doesn't raise `RuntimeError`. I think its very useful to have such mode.

**Comment by vkryachko** [comment]
Hi @renskiy,

Thanks for you interest in the library.

From what I understand you want to have a tasklocal that is valid outside a task. I think it is dangerous and harmful as it can lead to subtle and hard-to-track bugs.
That's why I intentionally made it so that the tasklocal fails loud and early instead of silently swallowing the error and returning a _global_ dict which can be a problem in itself as it is not thread-safe.

Can you elaborate a bit more as to why you think it would be useful and what your use-case is?

**Comment by renskiy** [comment]
@vkryachko, yes this can be dangerous and because of it strict mode is enabled by default.

My use-case is following:

```
class MyTaskLocals(tasklocals.local):
    def __init__(**kwargs):
        self.foo = 'bar'
```

Trying to get instance of this class outside of the task leads to `RuntimeError`
